app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -50,11 +47,12 @@ const AnalysisAvailable = ({
   analysis,
 }: {
   experiment: getExperiment_experimentBySlug;
-  analysis: AnalysisData | undefined;
+  analysis: AnalysisData;
 }) => {
   const slugUnderscored = experiment.slug.replace(/-/g, "_");
   return (
     <>
+      <MonitoringLink {...experiment} />
```

Comment:
Now that the summary page has its own monitoring link this was getting duplicated